modules/olm-understanding-operator-catalog-images.adoc

```diff
@@ -10,6 +10,28 @@ version of an Operator catalog. As of {product-title} 4.3, Red Hat-provided
 Operators are distributed via Quay App Registry catalogs from
 link:https://quay.io/[quay.io].
 
+.Red Hat-provided App Registry catalogs
+[cols="3,7",options="header"]
+|===
+|Catalog
+|Description
+
+|`redhat-operators`
+|Public catalog for Red Hat products packaged and shipped by Red Hat. Supported
+by Red Hat.
+
+|`certified-operators`
+|Public catalog for products from leading independent software vendors (ISVs).
+Red Hat partners with ISVs to package and ship. Supported by the ISV.
+
+|`community-operators`
+|Public catalog for software maintained by relevant representatives in the
+link:https://github.com/operator-framework/community-operators[operator-framework/community-operators]
+GitHub repository. No official support or guarantee of disconnected
+compatibility.
```

Comment:
Afaik the last part of that sentence is also true for certified operators, no? Not every single certified operator is guaranteed to mirror into a disconnected environment.

Reply:
I could add a "No guarantee of disconnected compatibility" but I suppose that is true of some in redhat-operators too. There's a link in the intro of this page to the KBase article that is currently chronicling which RH-provided Operators support disconnected: https://access.redhat.com/articles/4740011